pysys/utils/threadpool.py

```diff
@@ -85,6 +85,7 @@ def run(self):
 			if self._dismissed.isSet():
 				break
 			try:
+
```

Comment:
Intentional blank line? I'd have expected it before the `try:` if anywhere.

Reply:
fine, removed